frontend/javascripts/oxalis/view/layouting/tracing_layout_view.tsx

```diff
@@ -143,16 +139,20 @@ class TracingLayoutView extends React.PureComponent<PropsWithRouter, State> {
     this.setState({
       status: newStatus,
     });
-    // After the data is loaded recalculate the layout type and the active layout.
-    const { initialCommandType, viewMode, is2d } = this.props;
-    const layoutType = determineLayout(initialCommandType.type, viewMode, is2d);
-    const lastActiveLayoutName = getLastActiveLayout(layoutType);
-    const layout = getLayoutConfig(layoutType, lastActiveLayoutName);
-    this.setState({
-      activeLayoutName: lastActiveLayoutName,
-      model: layout,
-    });
-    initializeInputCatcherSizes();
+    if (newStatus == "loaded") {
```

Comment:
before this change, the code also ran when the status changed to "loading" which is not necessary as far as I see.